app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/InputWidgets/index.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { ChangeEvent } from 'react';
+import { IInputWidgetsProps } from 'components/WranglerGrid/SelectColumnPanel/DataTable/types';
+import { Checkbox, FormControlLabel, Radio } from '@material-ui/core';
+import styled from 'styled-components';
+
+const RadioInput = styled(Radio)`
+  &.MuiRadio-root {
+    padding: 0;
+  }
+`;
+
+export default function InputWidgets({
+  isSingleSelection,
+  selectedColumns,
+  handleSingleSelection,
+  columnDetail,
+  isCheckboxDisabled,
+  handleMultipleSelection,
+  columnIndex,
+}: IInputWidgetsProps) {
+  const isColumnSelected = selectedColumns.some((column) => column.label === columnDetail.label);
+  const disabledInputElement = !isColumnSelected && isCheckboxDisabled();
+
+  const onRadioInputChange = () => handleSingleSelection(columnDetail);
+  const onCheckBoxInputChange = (event) => handleMultipleSelection(event, columnDetail);
+
+  if (isSingleSelection) {
+    return (
+      <RadioInput
+        color="primary"
+        onChange={onRadioInputChange}
+        checked={isColumnSelected}
+        data-testid={`radio-input-${columnIndex}`}
+      />
+    );
+  } else {
+    return (
+      <FormControlLabel
```

Comment:
Why use a `FormControlLabel` with no label? And if it's needed, why not use it for the radio button?

Reply:
removed FormControlLabel and updated the code. FormControlLabel is not required as we do not have any labels attached to it.